web-client/integration-tests/docketClerkAddsTranscriptToDocketRecord.test.js

```diff
@@ -21,6 +18,7 @@ describe('Docket Clerk Adds Transcript to Docket Record', () => {
   const { TRANSCRIPT_EVENT_CODE } = applicationContext.getConstants();
 
   beforeAll(() => {
+    console.error = () => {};
```

Comment:
can this be removed?

Reply:
This was added to keep the console clean during the test execution.